### DISCUSSION TITLE
v2.22.0 ETA?

### DISCUSSION THREAD

**Discussion by gsg-nsb** [bootstrap-vue/bootstrap-vue#6573] [General]
Hello,

Is there any estimate as to when v2.22.0 will be ready for release? Or alternatively, would it be feasible to cut a prelease version?

I ask because #6283 has become an issue for us, and it's already fixed in the v2.22.0 project.

Thank you!

**Comment by Hiws** [comment]
Sorry, but there's currently no estimate when the next version will be released.

**Reply by gsg-nsb** [reply]
OK, thank you very much for the reply!

**Comment by scryptoking** [comment]
Is this project still alive? It's been almost 7 months since latest release...

**Reply by QiQiBiBi** [reply]
Guess it dying...
